utils/defillama.ts

```diff
@@ -1,68 +1,42 @@
 import axios from "axios";
 
-const baseUrl = "https://api.llama.fi/protocol/uma-protocol";
-const defillamaUrls: string[] = [
-  "https://api.llama.fi/protocol/polymarket",
-  "https://api.llama.fi/protocol/sherlock",
-  "https://api.llama.fi/protocol/jarvis-network",
-  "https://api.llama.fi/protocol/across",
-  "https://api.llama.fi/protocol/uma-protocol",
-];
+const baseUrl = "https://api.llama.fi/oracles";
+
+interface TvsData {
+  tvl: number;
+}
 
 interface TvlData {
   date: string;
   totalLiquidityUSD: number;
 }
 
-export async function getTvl(url: string = baseUrl) {
+export async function getDefillamaTvl(url: string = baseUrl) {
   const { data } = await axios.get(url);
-  const defillamaTvlData: TvlData[] = data.tvl.slice(-1);
-  const latestTvl = defillamaTvlData[0].totalLiquidityUSD;
-  return latestTvl;
-}
+  const defillamaTvlData: TvsData =
+    data.chart[Object.keys(data.chart)[Object.keys(data.chart).length - 1]][
```

Comment:
something doesnt look right here. are you trying to get the last value in the array or something?

Reply:
Yeah. The data is smth like
```
{ 
  chart: {
    [timestamp: string]: { 
      UMA: { ... }, 
      ... 
    } 
  },
  ...
}```